src/sentry/replays/consumers/recording/process_recording.py

```diff
@@ -124,6 +126,8 @@ def submit(self, message: Message[KafkaPayload]) -> None:
                     message,
                     current_transaction,
                 )
+            elif message_dict["type"] == "replay_recording_not_chunked":
+                ingest_recording_not_chunked(message_dict, current_transaction)
```

Comment:
any particular reason not to cast in this function here? differs from how the others are called